manager/manager/manager.go

```diff
@@ -183,7 +202,7 @@ func (m *manager) DeleteDeployment(name string, forget bool) (*Deployment, error
 	latest := getLatestManifest(d.Manifests)
 	if latest != nil {
 		log.Printf("Deleting resources from the latest manifest")
-		if err := m.deployer.DeleteConfiguration(latest.ExpandedConfig); err != nil {
+		if _, err := m.deployer.DeleteConfiguration(latest.ExpandedConfig); err != nil {
```

Comment:
If you set the argument forget to false, we do not actually delete the deployment from repository but mark it as deleted. So, seems safer to update the manifest in case it's being done. I could have added a check that doesn't set the manifest if forget = false.